src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+class Deadline extends Task {
+	private final LocalDate time;
```

Comment:
Perhaps this should be 4 spaces rather than a tab? I noticed you used tabs for indentation in some of your other classes too.